README.md

```diff
@@ -86,6 +86,7 @@
 - [Integrity](http://peacockmedia.software/mac/integrity/free.html) - Easily find your website's broken links.
 - [Paw](https://luckymarmot.com/paw) - The ultimate REST client.
 - [Insomnia](http://insomnia.rest) - A modern REST client with an beautiful interface.
+- [Postgres.app](http://postgresapp.com/) - The easiest way to get started with PostgreSQL. ![Open-Source Software][OSS Icon]
```

Comment:
@brunowego  The OSS icon should link to the Github repo, fix it and I'll merge :)